python_modules/dagster/dagster_tests/general_tests/utils_tests/test_cached_method.py

```diff
@@ -96,3 +91,35 @@ def stuff(self, a=None, b=None):
     assert a1 != b1
     assert a1 != a2
     assert b1 != b2
+
+
+def test_ordinal_args() -> None:
+    class MyClass:
+        @cached_method
+        def stuff(self, a, b) -> Dict:
+            return {"a": a, "b": b}
+
+    obj = MyClass()
+    a1 = obj.stuff(1, None)
+    b1 = obj.stuff(None, 1)
+    a2 = obj.stuff(2, None)
+    b2 = obj.stuff(None, 2)
+
+    assert a1 != b1
+    assert a1 != a2
+    assert b1 != b2
```

Comment:
Maybe this is covered by the test below, but I would have expected to see an assert like
```python
assert obj.stuff(1, None) is obj.stuff(a=1, b=None)
```

Reply:
Great call thx